core/src/test/scala/org/broadinstitute/dsde/rawls/entities/datarepo/DataRepoEntityProviderSpecSupport.scala

```diff
@@ -35,15 +36,21 @@ trait DataRepoEntityProviderSpecSupport {
   /* A "factory" method to create a DataRepoEntityProvider, with defaults.
    * Individual unit tests should call this to reduce boilerplate.
    */
-  def createTestProvider(workspaceManagerDAO: SpecWorkspaceManagerDAO = new SpecWorkspaceManagerDAO(Right(createDataRefDescription())),
```

Comment:
I know these have been around for a little bit, and weren't modified as part of this PR, but any interest in renaming the `Spec*` classes to `Mock*`?

Reply:
I am inclined to leave them alone. You can direct feedback to David An as I am not sure why he named them this way.